libs/common-services/src/test/java/uk/gov/di/ipv/core/library/domain/gpg45/Gpg45ProfileEvaluatorTest.java

```diff
@@ -343,7 +344,8 @@ void getCredentialByTypeShouldReturnCorrectType() throws Exception {
 
         JWTClaimsSet jwtClaimsSet = result.get().getJWTClaimsSet();
 
-        assertEquals("https://review-p.integration.account.gov.uk", jwtClaimsSet.getIssuer());
+        Assertions.assertEquals(
```

Comment:
I think we can just use the static import we've already got, so just `assertEquals()`

Reply:
done